pkgs/applications/audio/aaxtomp3/default.nix

```
@@ -36,17 +27,51 @@ stdenv.mkDerivation rec {
     hash = "sha256-7a9ZVvobWH/gPxa3cFiPL+vlu8h1Dxtcq0trm3HzlQg=";
   };
 
-  dontBuild = false;
+  # use whitespace to show osh arithmetic is not file redirection
+  patches = [./osh.patch];
 
-  nativeBuildInputs = [ makeWrapper ];
+  buildPhase = ''
```

```suggestion
  postPatch = ''
```

patching is always done in patchPhase. Please disable buildPhase again if there is nothing to build.